cmd/serverless-init/metric/metric.go

```diff
@@ -0,0 +1,28 @@
+package metric
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+)
+
+func ColdStart(tags []string, timestamp time.Time, demux aggregator.Demultiplexer) {
+	add("gcp.run.enhanced.cold_start", tags, time.Now(), demux)
+}
+
+func Shutdown(tags []string, timestamp time.Time, demux aggregator.Demultiplexer) {
```

Comment:
metric.Shutdown seems like an ambigous name, it wouldn't immediately jump into my mind that it's creating an shutdown enhanced metric.